### PR TITLE
fix(home-manager): link GTK 4.0 files

### LINKED ISSUE #107: GTK: link GTK 4.0 to correct places
In https://github.com/catppuccin/gtk it suggests that you link gtk 4.0 to given places. See the snippit below

```nix
xdg.configFile = {
  "gtk-4.0/assets".source = "${config.gtk.theme.package}/share/themes/${config.gtk.theme.name}/gtk-4.0/assets";
  "gtk-4.0/gtk.css".source = "${config.gtk.theme.package}/share/themes/${config.gtk.theme.name}/gtk-4.0/gtk.css";
  "gtk-4.0/gtk-dark.css".source = "${config.gtk.theme.package}/share/themes/${config.gtk.theme.name}/gtk-4.0/gtk-dark.css";
};
```

### PR DESCRIPTION
**Pull request by EnzioKam**
Adds links go GTK 4.0 files as suggested in the [catppuccin/gtk](https://github.com/catppuccin/gtk) README. Closes #107.

### CODE REVIEW

**Review by isabelroses — COMMENTED** [review]
you should include an xdg.enable assertion, some of the other themes have it.

**Comment by EnzioKam** [comment]
I added the assertion. Could you check if it is ok?

**Review by isabelroses — APPROVED** [review]

File: modules/home-manager/gtk.nix [hunk 1]
```diff
@@ -30,36 +30,48 @@ in
...
+
+    assertions = [
+      (lib.ctp.assertXdgEnabled "gtk")
+    ];
+
+    xdg.configFile = {
+      "gtk-4.0/assets".source = "${config.gtk.theme.package}/share/themes/${config.gtk.theme.name}/gtk-4.0/assets";
+      "gtk-4.0/gtk.css".source = "${config.gtk.theme.package}/share/themes/${config.gtk.theme.name}/gtk-4.0/gtk.css";
+      "gtk-4.0/gtk-dark.css".source = "${config.gtk.theme.package}/share/themes/${config.gtk.theme.name}/gtk-4.0/gtk-dark.css";
+    };
```
Comment (getchoo) [inline modules/home-manager/gtk.nix:75]:
```suggestion
    xdg.configFile = 
      let
        gtk4Dir = "${config.gtk.theme.package}/share/themes/${config.gtk.theme.name}/gtk-4.0";
      in
      {
      "gtk-4.0/assets".source = "${gtk4Dir}/assets";
      "gtk-4.0/gtk.css".source = "${gtk4Dir}/gtk.css";
      "gtk-4.0/gtk-dark.css".source = "${gtk4Dir}/gtk-dark.css";
    };
```

**Comment by EnzioKam** [comment]
Can you trigger the CI again? Had issues with nixpkgs-fmt earlier but should be fixed now.

**Review by getchoo — APPROVED** [review]
thanks for the fix! :)

**Pull request merged**